Model/Sync/Catalog/Processor.php

```diff
@@ -503,26 +462,21 @@ protected function processUnAssignData()
                     $returnResponse = $this->processResponse($params, $response, $tempDeleteQry, $itemData);
                 }
 
-                $sqlData = [];
-                $sqlData[] = $returnResponse['temp_sql'];
-                $this->insertOnDuplicate(
-                    'yotpo_product_sync',
-                    $sqlData
-                );
+                $this->updateSyncTable($returnResponse['temp_sql']);
             }
             $this->updateProductSyncLimit($dataCount);
         }
     }
 
     /**
      * @param array <mixed> $items
-     * @param boolean $visibleVariants
+     * @param boolean $isVariantsDataIncluded
      * @return array <mixed>
      * @throws NoSuchEntityException
      */
-    protected function manageSyncItems($items, $visibleVariants = false): array
+    protected function getSyncItems($items, $isVariantsDataIncluded): array
```

Comment:
@EranAvidor ,
Can we keep default declaration of $isVariantsDataIncluded = false here?

Reply:
This function is being [called once](https://github.com/YotpoLtd/magento2-module-core/blob/47637c75df2cdb3648347fd434de711b432a18a8/Model/Sync/Catalog/Processor.php#L225), and it passes the second argument.
Do we need a default value?